cefpanda/__init__.py

```diff
@@ -157,6 +172,9 @@ def __init__(self):
 
         self.use_mouse = True
 
+    def node(self):
```

Comment:
Do you have a good use case for exposing `_cef_node`? I was keeping this out of the "public" API in case I needed to change things under-the-hood. If we want to expose it, we might as well just make `_cef_node` public (i.e., rename it to something like `cef_node`).

Reply:
Any use case that does scene graph manipulations on the CEF element; movement, scaling, reparenting it...